### DISCUSSION TITLE
End Platform Obsidian Farm

### DISCUSSION THREAD

**Discussion by Liquify69420** [Stardust-Labs-MC/StardustMC#38] [Ideas]
I personally don't believe that it is fair for one individual to have complete control over a public area that can be used and abused to gain a almost monopoly on a resource (obsidian), I personally believe that since it is public property that no one should be allowed to use it for farms, or that they are only allowed to claim it when they have the farm running and that they aren't allowed to have the farm running for extended periods of time i.e. over something like 12 hours. But to allow one person to have complete control on such a valuable resource, that they can sell to most of the players is just unfair in my opinion. I think that the best solution is to have the platform admin claimed so that people can still enter but cannot use it to farm obsidian.

Thanks for listening to my TED Talk, Liquifiy (Lix9)

**Comment by aisukuma** [comment]
I believe Calm said that he's going to make the farm be available for everyone to use
